platform/windows/detect.py

```diff
@@ -169,7 +169,7 @@ def get_flags():
 
 	return [
 		('freetype','builtin'), #use builtin freetype
-		('openssl','builtin'), #use builtin openssl
+		#('openssl','builtin'), #use builtin openssl
```

Comment:
Nice for the trailing spaces, but you added this unwanted change + a cmd.exe shortcut.
I'd advise you to remove this last commit with: ``git rebase -i HEAD~2`` and remove the line of the last commit.

Then redo it while being cautious about what you ``git add``. Then, instead of making a new commit, you can use ``git commit --amend`` to squash those changes directly in the parent commit (i.e. your first commit in this PR).